Windows installation wiki + config, batch file (working on FTS 1.7.5,…

Cale-Torino opened the pull request:
… UI v1.5.10)

Windows version tested on FTS 1.7.5 and it's working.
Also added a few enhancements to the md

* Uninstall wiki.
* tried to make paths more user friendly added config files for easy copy pasting.
* requirements wiki for quick install.
* using /D switch in the batch file in order to run the script from another drive.
* some punctuation fixes etc.
* Added `config.py`, `MainConfig.py`, `StartFreeTAKServer.bat` and `requirements.txt` files

LargoUsagi commented:
How does this differ from this? https://github.com/FreeTAKTeam/FreeTAKServer-User-Docs/blob/main/docs/docs/Installation/Windows/FreeTAKServer_Windows_Install.md

Should these files be merged together?

Cale-Torino commented:
Yes, I removed some of the unnecessary links and paragraphs I left in when I uploaded the file on discord. Also added some more files and improvements to the commit.

Cale-Torino commented:
Tested this exact md on a fresh windows 10 machine and to my knowledge its running fine as of yesterday.
